pypeit/flatfield.py

```diff
@@ -262,16 +306,61 @@ def fit2illumflat(self, slits, frametype='illum', initial=False, flexure_shift=N
                 continue
             # Skip those without a bspline
             # DO it
-            _slitid_img = slits.slit_img(slitidx=slit_idx, initial=initial, flexure=flexure_shift)
+            _slitid_img = slits.slit_img(slitidx=slit_idx, initial=initial, flexure=spat_flexure)
             onslit = _slitid_img == slits.spat_id[slit_idx]
             spat_coo = slits.spatial_coordinate_image(slitidx=slit_idx,
                                                       initial=initial,
                                                       slitid_img=_slitid_img,
-                                                      flexure_shift=flexure_shift)
+                                                      flexure_shift=spat_flexure)
             illumflat[onslit] = spat_bsplines[slit_idx].value(spat_coo[onslit])[0]
         # TODO -- Update the internal one?  Or remove it altogether??
         return illumflat
 
+    def fit2illumflat_fine(self, slits, tilts=None, initial=False, spat_flexure=None, frametype='illum'):
+        """
+        Main algorithm to build the wavelength image
```

Comment:
wavelength image?

Reply:
Oops - thanks for spotting that - fixed!